contributing.md

```diff
@@ -2,17 +2,18 @@
 # Contributing
 
 ### Terms
-  * Upstream: the repository from the ```original_author```
-  * Origin: the forked repository under your ```account```
+<<<<<<< HEAD
```

Comment:
Hey, you missed deleting these lines after resolving conflicts.

Reply:
@sara-02 this is weird everything looks fine locally and the `git diff` check was accurate. Can you let me know if it's okay now.